feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/quicksettings/QuickSettingsScreen.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2024 The Android Open Source Project
+ * Copyright (C) 2023 The Android Open Source Project
```

Comment:
```suggestion
 * Copyright (C) 2024 The Android Open Source Project
```

Does it need to change years?

Reply:
I don't think so. I think the year was supposed to reflect the date the file was created, not last edited